libs/community/langchain_community/embeddings/infinity.py

```diff
@@ -43,6 +43,11 @@ class InfinityEmbeddings(BaseModel, Embeddings):
     client: Any = None  #: :meta private:
     """Infinity client."""
 
+    embed_instruction: str = "passage: "
```

Comment:
Should default to ""

Reply:
default to "" for document and also query ?

Reply:
For BGE, the instruction for query is `Represent this sentence for searching relevant passages:` 
https://huggingface.co/BAAI/bge-large-en-v1.5

Reply:
Incorrect, see their repo: `update embedding model: release bge-*-v1.5 embedding model to alleviate the issue of the similarity distribution, and enhance its retrieval ability without instruction.` - by adding random instruction very deep in langchain, users of langchain will become frustrated, as they do not understand why their retrieval accuracy dropped. 

Please remove both instructions, thank you.

Reply:
Thanks for the explanation, I set the defaults to ""